aptos-move/framework/aptos-stdlib/sources/math_fixed.move

```diff
@@ -0,0 +1,111 @@
+/// Standard math utilities missing in the Move Language.
+module aptos_std::math_fixed {
+    use std::fixed_point32;
+    use std::fixed_point32::FixedPoint32;
+
+    const EOVERFLOW: u64 = 1;
+
+    // Square root of fixed point number
+    public fun sqrt(x: FixedPoint32): FixedPoint32 {
+        let y = (fixed_point32::get_raw_value(x) as u128);
+        let prev_res = 0;
+        let res = y / 2;
+        while (res != prev_res) {
+            prev_res = res;
+            res = (res + (y << 32) / res) / 2;
+        };
+        fixed_point32::create_from_raw_value((res as u64))
+    }
+
+    // Exponent function with a precission of 6 digits.
+    public fun exp(x: FixedPoint32): FixedPoint32 {
+        fixed_point32::create_from_raw_value((exp_raw((fixed_point32::get_raw_value(x) as u128)) as u64))
+    }
+
+    // Integer power of a fixed point number
+    public fun pow(x: FixedPoint32, n: u64): FixedPoint32 {
+        fixed_point32::create_from_raw_value((pow_raw((fixed_point32::get_raw_value(x) as u128), (n as u128)) as u64))
+    }
+
+    // Specialized function for x * y / z that omits intermediate shifting
+    public fun mul_div(x: FixedPoint32, y: FixedPoint32, z: FixedPoint32): FixedPoint32 {
+        let a = (fixed_point32::get_raw_value(x) as u128);
+        let b = (fixed_point32::get_raw_value(y) as u128);
+        let c = (fixed_point32::get_raw_value(z) as u128);
+        fixed_point32::create_from_raw_value (((a * b / c) as u64))
```

Comment:
May be we can add math_u128::mul_div in this same PR and use it here?

Reply:
Why is mul_div useful for u128? I don't see why it's easier/more readable/nicer to write mul_div(a,b,c) over a*b/c

Reply:
I meant muldiv for u64 numbers. We need that function anyway so might as well add it here so this can call that?